addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -157,11 +155,11 @@ export class DiscussCoreCommon {
                 ) {
                     this.threadService.updateSeen(channel, last_message_id);
                 }
-                const seenInfo = channel.seenInfos.find(
-                    (seenInfo) => seenInfo.partner.id === partner_id
+                const member = channel.channelMembers.find(
+                    (m) => m.persona.id === partner_id && m.persona.type === "partner"
```

Comment:
send id in payload, this should eventually just become insert on channel member and not a specific notif